### PR TITLE
docs: Add Cloud Docs

### PR DESCRIPTION
**Pull request by tayloramurphy**
This takes the content from https://github.com/meltano/cloud-docs and pulls it into the main docs.

### CODE REVIEW

**Comment by WillDaSilva** [comment]
@tayloramurphy Will we be archiving https://github.com/meltano/cloud-docs after this has been merged?

**Comment by tayloramurphy** [comment]
@WillDaSilva yes. No need for the duplicate content. I can handle moving the PRs and any relevant issues (there's not many!)

File: docs/src/_cloud/api_reference.md
[the remark this answers is not in the document]
Comment (tayloramurphy) [reply docs/src/_cloud/api_reference.md:1]: @edgarrmondragon feel free to make a PR or suggestion to delete whatever - I'm just copying from the existing cloud-docs.

**Comment by tayloramurphy** [comment]
> can we take an iteration to remove/replace stale content before merging?

@aaronsteers absolutely!

**Comment by tayloramurphy** [comment]
@aaronsteers I added an info warning to the top of every page.

**Review by WillDaSilva — APPROVED** [review]
I'm approving this before I consider it stable so that this can be published prior to a meeting in which it will be shared.

Follow-up PRs should promptly address:
- [ ] Update mentions of `kms-ext`, which I do not believe is being used now that `meltano cloud config` can set secrets:
  - https://deploy-preview-7428--meltano.netlify.app/cloud/platform#ssh-private-keys-for-private-git-repository-package-access
  - https://deploy-preview-7428--meltano.netlify.app/cloud/encrypting_secrets
  - https://deploy-preview-7428--meltano.netlify.app/cloud/onboarding#step-2-encrypt-and-submit-the-env-file
- [ ] Remove alpha-only instructions to grant access to the GitHub accounts `@MeltanoCloud`, `@WillDaSilva`, `@magreenbaum`, and `@kgpayne`
  - https://deploy-preview-7428--meltano.netlify.app/cloud/onboarding#prereq-1-provide-access-to-your-repo

**Comment by tayloramurphy** [comment]
Merging!

**Pull request merged**